workflow-templates/im-test-k6-manual-advanced.yml

```diff
@@ -0,0 +1,66 @@
+name: Manual trigger K6 test - Advanced
+
+on: 
+  workflow_dispatch:
+    inputs:
+      envToTest:
+        description: 'Environment to run test against [[1] dev, [2] qa, [3] stage]'   
+        required: true
+        default: '1'
+      testFolderRoot:
+        description: 'Test folder root directory from root of repo'
+        required: true
+        default: "tests/k6"
```

Comment:
For this you should add a TODO: for users to verify whether this is correct for their app.  Most of the items where you provide a value for the default should have that TODO.

Reply:
Can you suggest an edit to show me the format of the TODO and where to put it?

Reply:
added todo